test-case-reporting/index.ts

```diff
@@ -40,47 +60,33 @@ app.get('/test-results/build/:buildNumber', async (req, res) => {
 
   const pageSize = 100;
 
-  const testRunJson = {
-    testRuns: await testResultRecord.getTestRunsOfBuild(buildNumber, {
-      limit: pageSize,
-      offset: 0,
-    }),
-  };
+  const testRuns = await testResultRecord.getTestRunsOfBuild(buildNumber, {
```

Comment:
```javascript
  const testRuns = await testResultRecord.getTestRunsOfBuild(buildNumber, extractPageInfo(req));
```

Reply:
Done, I believe (done in commit https://github.com/ampproject/amp-github-apps/pull/943/commits/2e25956e8a6deca14e692053161ff4ef4cae607a)